Tools/autotest/common.py

```diff
@@ -10771,6 +10847,82 @@ def test_ltm(self):
                     self.progress("  Fulfilled")
                     del wants[want]
 
+    def convertDdToDmsFormat(self, ddm):
+        '''converts decimal degree values to degree minutes'''
+        deg = math.trunc(ddm / 1e7)
+        mm = (deg * 100 + ((ddm / 1e7) - deg) * 60) / 100
+        if mm < -180.0 or mm > 180.0:
+            mm = 0.0
+        return math.trunc(mm * 1.0e7)
+
+    def test_devo(self):
```

Comment:
```suggestion
    def DEVO(self):
```

Trying to be more consistent about making these methods the same name as the test.

Reply:
Done!